builder/assets/clever.service

```diff
@@ -1,12 +1,11 @@
 [Unit]
 Description=Clever ROS package
-Requires=roscore.service
-After=roscore.service
+After=network.target
 
 [Service]
 User=pi
-EnvironmentFile=/lib/systemd/system/roscore.env
-ExecStart=/opt/ros/kinetic/bin/roslaunch clever clever.launch --wait --screen
+ExecStart=/bin/bash -c ". /home/parallels/catkin_ws/devel/setup.bash; \
```

Comment:
Please correct the path here. Also, why are we using `bash` for clever and `sh` for roscore? FYI, `setup.bash` basically sources `setup.sh` in ROS Kinetic.

Reply:
I guess, we should use `setup.sh` in both .service-files?

Reply:
Yes, we should prefer using `sh` scripts wherever possible, they seem to have better compatibility.